### PR TITLE
[LLHD] Replace llhd.neg with comb.sub (#1654)

### LINKED ISSUE #1654: [LLHD] Replace `llhd.neg` with `comb.sub`
Replace `llhd.neg x` with `comb.sub 0, x`.

### PR DESCRIPTION
**Pull request by anurudhp**
Fixes #1654

### CODE REVIEW

**Review by maerhart — APPROVED** [review]
Looks good to me! Thank you for contributing :smile:

File: test/Dialect/LLHD/Export/verilog_arithmetic.mlir [hunk 1]
```diff
@@ -4,8 +4,10 @@
 llhd.entity @check_arithmetic() -> () {
   // CHECK-NEXT: wire [63:0] _[[A:.*]] = 64'd42;
   %a = llhd.const 42 : i64
-  // CHECK-NEXT: wire [63:0] _{{.*}} = -_[[A]];
-  %0 = llhd.neg %a : i64
+  // CHECK-NEXT: wire [63:0] _[[Z:.*]] = 64'd0;
+  %z = llhd.const 0 : i64
+  // CHECK-NEXT: wire [63:0] _{{.*}} = _[[Z]] - _[[A]];
+  %0 = comb.sub %z, %a : i64
```
[the remark this answers is not in the document]
Comment (anurudhp) [reply test/Dialect/LLHD/Export/verilog_arithmetic.mlir]: Ah, right! Removed it now.

[the remark this answers is not in the document]
Comment (maerhart) [reply test/Dialect/LLHD/Export/verilog_arithmetic.mlir]: Thanks! Do you have commit access or should I hit merge for you?

[the remark this answers is not in the document]
Comment (anurudhp) [reply test/Dialect/LLHD/Export/verilog_arithmetic.mlir]: Nope, I don't have commit access. Could you merge it? thanks!

**Pull request merged**